src/test/suite/fixtures/recorded/actions/clearThat2.yml

```diff
@@ -0,0 +1,30 @@
+languageId: plaintext
```

Comment:
As above not sure how this differs from other "that" test

Reply:
comment about source